### PR TITLE
Handle fixed statement in control flow graph

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@mavasani, @cston, @dotnet/analyzer-ioperation, @dotnet/roslyn-compiler Please review

**Comment by AlekseyTs** [comment]
@mavasani, @cston, @dotnet/analyzer-ioperation, @dotnet/roslyn-compiler Please review

**Review by jcouv — APPROVED** [review]
LGTM

**Review by mavasani — APPROVED** [review]

**Comment by AlekseyTs** [comment]
@cston, @dotnet/roslyn-compiler Please review, need a second review.

**Comment by AlekseyTs** [comment]
@cston, @dotnet/roslyn-compiler Please review, need a second review.

**Review by 333fred — APPROVED** [review]
LGTM other than the extra verification.

File: src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs
Comment (AlekseyTs) [reply src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs]: > Can we verify that all these symbols have IsFixed set correctly?

That wouldn't be the right thing to verify. They don't have to be. The right behavior of IsFixed API is verified in other unit-tests, not specific to CFG

---
In reply to: [180186139](https://github.com/dotnet/roslyn/pull/25974#discussion_r180186139) [](ancestors = 180186139)

**Pull request merged**
